radio/src/translations.cpp

```diff
@@ -121,6 +121,10 @@ ISTR(TEXT_SIZE);
 ISTR(SUBTRIMMODES);
 ISTR(PWR_OFF_DELAYS);
 
+const char STR_MINUTE_SINGULAR[] = TR_MINUTE_SINGULAR;
+const char STR_MINUTE_PLURAL1[] = TR_MINUTE_PLURAL1;
+const char STR_MINUTE_PLURAL2[] = TR_MINUTE_PLURAL2;
```

Comment:
PLURAL2 seems to be defined only for English. What is it actually meant for? Are their languages with different plurals?